xain_fl/fl/coordinator/aggregate.py

```diff
@@ -18,17 +18,15 @@ def __init__(self):
         """Initialize the aggregator."""
 
     @abstractmethod
-    def aggregate(
-        self, mult_model_weights: List[List[ndarray]], aggregation_data: List[int]
-    ) -> List[ndarray]:
+    def aggregate(self, mult_model_weights: List[ndarray], aggregation_data: List[int]) -> ndarray:
         """Aggregate the weights of multiple models.
 
         Args:
-            mult_model_weights (List[List[ndarray]]): The weights of multiple models.
+            mult_model_weights (List[ndarray]): The weights of multiple models.
```

Comment:
i guess `multiple_model_weights` would be more explicit. it only saves 4 characters afterall

Reply:
i thought about this as well, fine with both variants, i'll change it accordingly.

Reply:
changed.